helm-charts/azure-api-management-gateway/templates/deployment.yaml

```diff
@@ -24,6 +24,7 @@ spec:
         dapr.io/log-level: {{ .Values.dapr.logging.level | quote }}
       {{- end }}
     spec:
+      terminationGracePeriodSeconds: {{ .Values.gateway.deployment.terminationGracePeriodSeconds }}
```

Comment:
We should check if it's defined, otherwise don't add it to use Kubernetes defaults

Reply:
```suggestion
       {{- if .Values.gateway.deployment.terminationGracePeriodSeconds }}
      terminationGracePeriodSeconds: {{ .Values.gateway.deployment.terminationGracePeriodSeconds }}
      {{- end }}
```